packages/flutter_test/test/controller_test.dart

```diff
@@ -805,4 +675,278 @@ void main() {
       expect(find.text('Item b-45'), findsOneWidget);
     });
   });
+
+  group('SemanticsController', () {
+    group('get', () {
+      testWidgets('throws when there are no semantics', (WidgetTester tester) async {
+        await tester.pumpWidget(
+          const MaterialApp(
+            home: Scaffold(
+              body: Text('hello'),
+            ),
+          ),
+        );
+
+        expect(() => tester.semantics.get(find.text('hello')), throwsStateError);
+      }, semanticsEnabled: false);
+
+      testWidgets('throws when there are multiple results from the finder', (WidgetTester tester) async {
+        final SemanticsHandle semanticsHandle = tester.ensureSemantics();
```

Comment:
nit: do you need this? I thought `semanticsEnabled` on testWidgets defaults to true and means this is no longer requried.

Reply:
here and also for the other tests.

Reply:
Looks like you're correct! Most of the other semantics tests I've looked add had that call in place, so I assumed. I may write up a ticket to go clean those up so our code is up to date with the latest patterns elsewhere.

Reply:
Done for all of the semantics tests in this file.